.github/workflows/ci.yml

```
@@ -3,6 +3,7 @@ name: CI
 on:
   push:
     branches: [main]
+    branches: [main]
```

actionlint (1.7.12)

[error] key "branches" is duplicated in "push" section. previously defined at line:5,col:5

(syntax-check)